ttnn/cpp/ttnn/multi_device.hpp

```diff
@@ -65,25 +63,33 @@ Tensor aggregate_as_tensor(std::vector<Tensor>& tensor_shards)
     if (storage_type == StorageType::OWNED) {
         std::vector<tt::tt_metal::Shape> shapes;
         std::vector<OwnedBuffer> host_owned_buffers;
-        for (const auto &shard : tensor_shards) {
+        for (const auto& shard : tensor_shards) {
             host_owned_buffers.push_back(std::get<OwnedStorage>(shard.get_storage()).buffer);
             shapes.push_back(shard.get_legacy_shape());
         }
         auto storage = MultiDeviceHostStorage{AllGatherTensor(), std::move(host_owned_buffers), shapes};
-        return Tensor(std::move(storage), tensor_shards.at(0).get_legacy_shape(), tensor_shards.at(0).get_dtype(),  tensor_shards.at(0).get_layout());
+        return Tensor(
```

Comment:
I prefer first arg to be on the same line, rest of args aligned with the first one. 
Same for function decls as it makes things easy to read when opening { is on the same line, else function args and function body are hard to distinguish